extensions/amp-analytics/amp-analytics.md

```diff
@@ -222,12 +222,24 @@ As an example, the following configuration can be used to sample 50% of the requ
 },
 ```
 
+
+#####  Element selector
+
+Some triggers such as `click` and `visible` allow specifying an single element or a collection of elements using the selector properties. Different triggers can apply different limitations and interpretations on selected elements, such as whether a selector applies to all matched element or the first one, or which elements can be matched: all or only AMP elements. See the documentation of each relevant trigger for more details.
```

Comment:
This is basically "see the documentation for each relevant trigger". In other words, the interpretation is up to the trigger. I.e. click matches all elements, while visibility only apply to the first found. Some of these also arbitrary: the only reason we don't allow non-AMP element visibility is because our InOb polyfills are not yet ready.